doc/manual/en/config/widgets/page/index.rst

```diff
@@ -43,6 +43,57 @@ Allowed attributes in the Page-element
         <text><label>Test</label></text>
     </page>
 
+2D pages
+^^^^^^^^
+
+At 2D pages it is possible to freely position the widgets on the page as well
+as use a background image. A typical use case is a floor plan as backdrop
+where in each room the relevant widgets are positioned accordingly.
+
+Background image
+................
+
+The size of the background in relation to the browser window is controlled
+by the attribute ``size``. The following holds:
+
+.. figure:: _static/size.svg
+    :alt: size attribute
+
+| The attribute ``backdropalign`` can move the position of the background
+  image. By default it is centered, which is the same as a value of ``50% 50%``.
+| Setting ``0px 50%`` would place it on the left side but vertical centered and
+  ``0px 0px`` would place it on the left and top of the page.
+
+Widgets
+.......
+
+For the widgets it is important to set the ``x`` and ``y`` attributes of its
+``layout`` element. These values must have a CSS unit. Typical would be a
+value like ``12%`` for a relative position and ``123px`` for an absolute
+position in pixel.
+
+.. tip::
+
+    To figure out the optimal value for the position it is possible to use
+    the development mode of the modern browsers: this mode is activated by
+    pressing :kbd:`ctrl` + :kbd:`shift` + :kbd:`i`.
+
+    First the widget must be roughly positioned in the config file, then this
+    config must be opened in the browser where the development mode must be
+    activated. Then the "box with cursor" must be clicked followed by a click
+    on the widget of interest in the browser window itself. This widget is now
+    shown in the DOM tree. Probably it is necessary to go there a few levels up
```

Comment:
Sounds wrong -> "..it is necessary to go a few levels up there" sounds better to me.